sample-notificator-domain/src/main/groovy/life/qbic/business/notification/create/CreateNotification.groovy

```diff
@@ -28,8 +28,13 @@ class CreateNotification implements CreateNotificationInput, FetchSubscriberOutp
     @Override
     void createNotifications(String date) {
         fetchSubscriberInput.fetchSubscriber(date)
-        //assume notificationPerSubscriber will be filled by the output method fetchedSubscribers of the FetchSubscriber use case
-        output.createdNotifications(notificationPerSubscriber)
+        //ToDo Remove this dummy data after testing since this will be provided by the use case
+        List<NotificationContent> notificationContentList = []
+        NotificationContent notificationContent1 = new NotificationContent.Builder("Jo", "My_Dude", "Jo.mydude@coolguy.de", "coolProject", "NICE1", 2, 1000).build()
+        NotificationContent notificationContent2 = new NotificationContent.Builder("Yas", "Queen", "Yas.queen@slaythem.de", "sweetProject", "COLD1", 1, 12).build()
```

Comment:
:D I like your examples

Reply:
please delete it before merging to the use case :)

Reply:
Will do thanks for the headsup!